blinkit-app/src/homePage/atoms/insertSelect/InsertSelect.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import { UserConsumer } from "../../../userContext";
+
+const addOption = (value, index) => {
```

Comment:
const addOption = ({key,name}, index) => {<option key={key} value={name} hidden>

use a unique relevant key for every option instead of giving/generating generic keys, which makes it easier to differentiate the options among themselves